.github/workflows/kind-e2e.yaml

```diff
@@ -35,18 +39,46 @@ jobs:
       run: |
         make toolchain
         make install-kwok
-        KWOK_REPO=kind.local KIND_CLUSTER_NAME=chart-testing make apply-with-kind
+        export KWOK_REPO=kind.local
+        export KIND_CLUSTER_NAME=chart-testing
+        make apply-with-kind
     - name: ping cluster
       shell: bash
       run: | 
+        sleep 15
```

Comment:
Any reason to add the 15s in here?

Reply:
Yes, if you start the KPIs too fast then you might not have Karpenter metrics getting scraped by Prometheus yet, waiting a short bit prevents problems with expecting metrics to exist that don't exist yet.